src/renderer/chat/ChatTab.ts

```diff
@@ -246,6 +247,17 @@ export default class ChatTab {
       if (this.textAreaElt.value.length > 0) return;
       this.textAreaElt.value = this.previousMessage;
       event.preventDefault();
+    }else if(event.keyCode == 9 /* TAB */){        
+        let toComplete = "";
+        for(let i = this.textAreaElt.selectionStart - 1; i >= 0; --i){
+            if(this.textAreaElt.value.charAt(i) == " ") break;

```

Comment:
Simplified this to use `.lastIndexOf` in b7270e85e018b5030d174d84252b780c024166a8.